app-sso-client/README.md

```diff
@@ -0,0 +1,59 @@
+# AppSSO Client Registration Fragment
+
+A Fragment that include AppSSO Client Configuration and Workload reference into your Host Accelerator. This enables
+a Workload to support authentication delegated to an AppSSO AuthServer.
+
+It generates `client-registration.yaml` and related `client-registration-claim.yaml` to be created at your workload namespace
+
+It changes your `workload.yaml` to reference the Client Registration Claim: this will bind the Client Registration into your workload.
+
+> Currently `workload.yaml` modification works ONLY if there is no other service claims defined in it!
+
+## Using the fragment
+
+To use this fragment in your accelerator add the following import under the `accelerator` section in your `accelerator.yaml`:
+
+```
+accelerator:
+
+# ...
+
+  imports:
+  - name: app-sso-client
+```
+
+Then in your `engine` section add a top level `chain` entry with a `merge` that contains your original engine processing
+transforms. This should then be followed by another `merge` that invokes the fragment processing, taking the files
+generated in the previous step as input.
+
+```
+engine:
+  chain:
```

Comment:
This seems too directive (and a bit weird)

A typical accelerator uses a top-level `merge`. Given what this fragment does, I would expect that common usage looks like:

Before
```
engine:
  merge:
  - T1
  - T2
```

After
```
engine:
  merge:
  - T1
  - T2
  - type: InvokeFragment
     reference: app-sso-client
```

But my general point is that this README should be less directive in its tone (eg using sentences like "for example") and inform users how the fragment works so that they could adapt the use to their usage if they need to.

Reply:
Done